tests/TestCompose_of-of/docker-compose.yml

```diff
@@ -10,7 +10,7 @@ services:
     command: >
       /bin/bash -c "source /opt/openfoam4/etc/bashrc && 
       cd /home/precice/openfoam-adapter/tutorials/CHT/flow-over-plate/buoyantPimpleFoam-laplacianFoam &&
-      sed -i 's|gather-scatter\"|gather-scatter\" exchange-directory=\"/home/precice/Data/Exchange/\" network=\"eth0\"|g' precice-config_serial.xml &&
+      sed -i 's|<m2n:sockets from=\"Fluid\" to=\"Solid\"/>|<m2n:sockets from=\"Fluid\" to=\"Solid\" exchange-directory=\"/home/precice/Data/Exchange/\" network=\"eth0\"/>|g' precice-config.xml &&
```

Comment:
It might be due to the fact that the sed line insertion that defines the network fails. That pattern doesn't exist in the tutorial yet (which is from the docker image openfoam-adapter-ubuntu1604.home-develop:latest)